unreleased.rst

```diff
@@ -14,4 +14,7 @@ HEAD — ongoing
    * Enable the ability to configure ledger api servers with a time service (for test purposes only).
    * Allow a ledger api server to share the DAML engine with the DAML-on-X participant node for performance. See `issue #2975 <https://github.com/digital-asset/daml/issues/2975>`__.
    * Allow non-alphanumeric characters in ledger api server participant ids (space, colon, hash, slash, dot).
-   * Include SQL statement type in ledger api server logging of SQL errors.
\ No newline at end of file
+   * Include SQL statement type in ledger api server logging of SQL errors.
```

Comment:
Is this a merge artifact?

Reply:
I think this just shows up because I added a trailing newline.

Reply:
Ah, thanks got it, just looked weird in the diff view.